### PR TITLE
 refactor: repo scripts section #197

### LINKED ISSUE #197: refactor: repo scripts section
### Describe the Feature Request

We want to make it easy for the command line to deploy the contracts in the repo to facilitate activities. This is part of a usability upgrade when running the `swaplace-subgraph` locally.

- [ ] Change the `deploy` script name into `deploy:swaplace` 
- [ ] Add another script as `deploy:mocks` with the deployMoks.ts script
- [ ] Remove the testnet script as it is ambiguous when simply using `yarn test --network $1`

### Describe Preferred Solution

```
/* Remove */
 "testnet": "npx hardhat test --network $1",

/* Refactor */
 "deploy": "npx hardhat run scripts/deploy.ts --network $1",

 ---

 "deploy:swaplace": "npx hardhat run scripts/deploy.ts --network $1",
 "deploy:mocks": "npx hardhat run scripts/deployMocks.ts --network $1",
```

### Related Code

[package.json](https://github.com/blockful-io/swaplace-contracts/blob/main/package.json)

### PR DESCRIPTION
**Pull request by blackbeard002**
closes #197

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,10 +30,10 @@
...
     "test": "npx hardhat test",
     "docs": "npx hardhat docgen",
-    "testnet": "npx hardhat test --network $1",
-    "deploy": "npx hardhat run scripts/deploy.ts --network $1",
+    "deploy:swaplace": "npx hardhat run scripts/deploy.ts --network $1",
     "compile-echidna": "crytic-compile . && slither . --print echidna",
     "fuzz-p": "echidna . --contract TestSwaplace --test-mode property --config echidna.config.yml",
-    "fuzz-a": "echidna . --contract TestSwaplace --test-mode assertion --config echidna.config.yml"
+    "fuzz-a": "echidna . --contract TestSwaplace --test-mode assertion --config echidna.config.yml",
+    "deploy:mocks": "npx hardhat run scripts/deployMock.ts --network $1"
```
Comment (0xneves) [inline package.json:37]: Hey @blackbeard002 Could you please move line 37 before line 33 so that the scripts stay ordered by their category/alphabetic

**Review by 0xneves — CHANGES_REQUESTED** [review]
Minor request ;)

File: package.json (hunk 1 above)
Comment (blackbeard002) [reply package.json:37]: Aye aye captain, I'm on it

**Comment by blackbeard002** [comment]
Hey @0xneves made the changes as requested

**Review by 0xneves — APPROVED** [review]
Awesome Deepu!! Approving the changes! I will ask you to submit to develop from now on! Thank you

**Pull request merged**